wpiunits/src/main/java/edu/wpi/first/units/Per.java

```diff
@@ -10,10 +10,8 @@
 /**
  * Generic combinatory unit type that represents the proportion of one unit to another, such as
  * Meters per Second or Radians per Celsius.
- *
- * <p>Note that due to restrictions with the Java type system, velocities (change per unit time) are
- * represented by the {@link Velocity} class. Accelerations are represented by {@code
- * Velocity<Velocity<X>>}, and so on.
```

Comment:
This comment mentions limitations of generics, but it was not descriptive enough to be meaningful. It was also brought up in frc-docs that the use of *X* in the example is confusing.
I simplified it to let users know they should use `Velocity` for velocity without going into details about why.